dali/pipeline/operators/crop/crop.cu

```diff
@@ -71,16 +71,16 @@ DALIError_t BatchedCrop(const uint8 *const *in_batch, const int *in_strides,
       C, H, W, in_batch, in_strides, L, out_batch, output_offsets);
   return DALISuccess;
 }
+
 }  // namespace
 
 template <>
 template <typename Out>
 void Crop<GPUBackend>::RunHelper(Workspace<GPUBackend> *ws, const int idx) {
   auto &output = ws->Output<GPUBackend>(idx);
-
   DALI_CALL((BatchedCrop<Out>(
       input_ptrs_gpu_.data<const uint8 *>(),
-      input_strides_gpu_.data<int>(), batch_size_,
+      input_strides_gpu_.data<int>(), batch_size_ * SequenceSize(idx),
```

Comment:
This is the part I have some doubts about: it assumes that all sequences in the batch have same length. I think is a strong assumption and one that might be confusing to our oursers: on one hand, each sample in the batch can have different size, but on the other, we impose uniform sequence length throughout the batch. It's far from obvious and probably we're going to see some complaints about it.

Reply:
For now, our Sequence-wise operators only produce unique-size sequences. In the future, we can think about extending the logic to by keeping the sequence-sizes per sample, but this would be subject for all another PR, and I don't think it's on our priority list for now.

However, I'll add an enforcement in order to fail if the user tries to feed multiple-size sequences with the external reader.

Reply:
Done